web/src/api/MercuryAPI.tsx

```diff
@@ -1,9 +1,10 @@
-import { Configuration, TodosApi } from './generated'
+import { ComponentApi, Configuration } from './generated'
 
-export class MercuryAPI extends TodosApi {
+// TODO: Combine all generated api's into MercuryAPI
+export class MercuryAPI extends ComponentApi {
   constructor(token: string) {
     const configuration = new Configuration({
-      basePath: 'http://localhost',
```

Comment:
Not sure I understand what this does and why it is/is not needed?

Reply:
its used as a prefix for every api request.
If it's set to localhost it will be that in radix as well. Which wont work.

if its an empty string, it will use the "where i'm being served from", which will be correct